polyply/src/apply_links.py

```diff
@@ -423,6 +447,11 @@ def run_molecule(self, meta_molecule):
         """
         molecule = meta_molecule.molecule
         force_field = meta_molecule.force_field
+        # we need to update the temporary interactions dict
+        self._update_applied_links(molecule.interactions, molecule, molecule.citations, mapping=None)
```

Comment:
Update temporary interactions dict, or update links? Is the method name wrong or the comment?

Reply:
I think the method's name should probably be update_temp_dict or something along those lines